### ISSUE TITLE
 Custom Mixer interface in the Config Tab

### ISSUE THREAD

**Issue by skaman82** [skaman82/cleanflight-configurator#54]
Something that works like this: http://www.iforce2d.net/mixercalc/

**Comment by NightHawk32** [comment]
Additional thoughts:
- drag and drop for the motors is not necessary
- instead: fields for the distances of the motors
- should be placed in the configuration tab --> mixer box
- new icon for custom mixer
- config file with mixers for popular frames

**Comment by skaman82** [comment]
How many motors should we offer to configure? I would start with 4. If necessary someone can add more motors later. Editing more than four motors makes this just more complicated to implement. What do you think?

**Comment by skaman82** [comment]
here some first thoughts as a simple sketch:

<img width="645" alt="bildschirmfoto 2015-12-05 um 16 20 43" src="https://cloud.githubusercontent.com/assets/11768024/11608525/4c795640-9b6c-11e5-969b-5c9e61158722.png">

**Comment by NightHawk32** [comment]
Yes, 4 motor should be the most common use case at the moment. We should get that working, more motors just make things too complicated for the beginning.
Your sktech looks awesome, great work!
This way it should me almost self-explaining how a custom mix is done, very nice.

**Comment by skaman82** [comment]
prepared the tab elements

<img width="508" alt="bildschirmfoto 2015-12-06 um 15 27 22" src="https://cloud.githubusercontent.com/assets/11768024/11613805/736603da-9c2e-11e5-8230-332d496277dc.png">
<img width="669" alt="bildschirmfoto 2015-12-06 um 15 27 43" src="https://cloud.githubusercontent.com/assets/11768024/11613806/7543e050-9c2e-11e5-8edb-568324798089.png">

**Comment by skaman82** [comment]
The Button "Edit Custom Mixer" should only appear when Custom Mixer is selected in the drop down menu

**Comment by skaman82** [comment]
useful link: http://stephanwagner.me/jBox (close event for the save button)

**Comment by Rob2160** [comment]
The interface looks good - consider also showing the CoT (optimum CG) position as a distance from front motors. (always mid point between front and rear motors in a quad - my mixer assumes this.)

However the location will vary for non symmetrical Hexacopters and V-Tails (Based on the V angle)

examples attached - CoT is automatically calculated for Hex and V-tails in the spreadsheet. 

![screen shot 2016-01-07 at 11 37 35 am](https://cloud.githubusercontent.com/assets/11248960/12159171/2ec1cb32-b534-11e5-9bdf-a744c1704d71.png)
![screen shot 2016-01-07 at 11 39 02 am](https://cloud.githubusercontent.com/assets/11248960/12159172/2f13b0fa-b534-11e5-8e8a-77350a07573b.png)
![screen shot 2016-01-07 at 11 41 44 am](https://cloud.githubusercontent.com/assets/11248960/12159173/2f1a7e3a-b534-11e5-94d8-6f2821fb46cd.png)
